diff -ruN /opt/x-tools/tos-esp-sdk/sdk/include/c_types.h sdk/include/c_types.h
--- sdk/include/c_types.h	2018-08-23 02:09:21.000000000 -0700
+++ sdk/include/c_types.h	2019-02-24 02:26:12.680307673 -0800
@@ -25,20 +25,7 @@
 #ifndef _C_TYPES_H_
 #define _C_TYPES_H_
 
-typedef unsigned char       uint8_t;
-typedef signed char         sint8_t;
-typedef signed char         int8_t;
-typedef unsigned short      uint16_t;
-typedef signed short        sint16_t;
-typedef signed short        int16_t;
-typedef unsigned int        uint32_t;
-typedef signed long         sint32_t;
-typedef signed int          int32_t;
-typedef signed long long    sint64_t;
-typedef unsigned long long  uint64_t;
-typedef unsigned long long  u_int64_t;
-typedef float               real32_t;
-typedef double              real64_t;
+#include <stdint.h>
 
 typedef unsigned char       uint8;
 typedef unsigned char       u8;
